qemu/tests/nic_hotplug.py

```diff
@@ -50,6 +49,9 @@ def run(test, params, env):
     nic_hotplugged = []
     try:
         for nic_index in xrange(1, nic_hotplug_count + 1):
+            # need to reconnect serial port after
+            # guest reboot for windows guest
+            session = vm.wait_for_serial_login(timeout=login_timeout)
```

Comment:
yes, we need as commented in above line that we have reboot steps in for loop, for windows guest serial_session will lost when guest reboot.

Thanks,
@xutian